### PR TITLE
Downloader app

### PR DESCRIPTION
**Pull request by Laurie-Bamber**
- Fixing CSS in downloader-app
- Updating init create_tables script

### CODE REVIEW

File: open-aps-db/init-scripts/8-create-openaps-data-tables.sql [hunk 1]
```diff
@@ -1,153 +1,146 @@
 
 CREATE TABLE openaps.device_status (
-  seq_id BIGSERIAL PRIMARY KEY,
```
Comment (ekeith-mudano) [inline open-aps-db/init-scripts/8-create-openaps-data-tables.sql]: Afraid I'm going to be a massive pain on this. Would it be possible to format the new lines so they match the format of the existing ones (2 spaces indent, no tabs)? This narrow the diffs down to just what has actually changed between versions, would be easy for something unintended to slip through when all lines are showing diffs.

File: open-aps-db/init-scripts/8-create-openaps-data-tables.sql [hunk 2]
```diff
@@ -178,11 +171,28 @@ CREATE TABLE openaps.member_demographics (
...
 GRANT SELECT ON TABLE openaps.member_demographics TO admin_viewer;
 
 
+CREATE TABLE openaps.source_entities (
+	
+	id BIGSERIAL PRIMARY KEY,
+	name VARCHAR,
+	inserted_ts TIMESTAMP
+	);
+ALTER TABLE openaps.source_entities OWNER TO power_user;
```
Comment (ekeith-mudano) [inline open-aps-db/init-scripts/8-create-openaps-data-tables.sql]: power_user doesn't exist in this database, these scripts will automatically be ran by the db admin user and so all tables already belong to that user. please remove.

File: open-humans-etl/setup/tables.sql [hunk 3]
```diff
@@ -1,19 +1,16 @@
```
Comment (ekeith-mudano) [inline open-humans-etl/setup/tables.sql]: this file is just for record keeping until we're happy the new scripts are finalised, at which poitn we can remove this. For this reason haven't reviewed this, if you're happy with it then I am.

Comment (Laurie-Bamber) [reply open-humans-etl/setup/tables.sql]: Sounds good to me.

**Pull request merged**